### DISCUSSION TITLE
Post Session Stewards Decisions

### DISCUSSION THREAD

**Discussion by andytratter** [theOehrly/Fast-F1#639] [Q&A] [answered]
Hello all,

How does FastF1 handle post session stewards decisions that have an outcome on results? Like post race penalties? Is the data release delayed enough to take these into account or does it update once results are finalised/official?

Thanks!

**Comment by Casper-Guo** [comment, marked as answer]
The data update time provides no guarantee with regards to stewards' decisions. IIRC ErgastAPI is used as the data source for official classification and there is no guarantee for when their data is finalized.

For example, the decision to disqualify LEC and HAM following the USGP last year was a bit delayed. The official classification data you can load now is correct but that wasn't the case immediately following the race, even if the timing data was available

**Reply by andytratter** [reply]
Ok that makes sense, thanks!
